volunteer/models/volunteer_shift_recurrent_subscription.py

```diff
@@ -0,0 +1,42 @@
+# SPDX-FileCopyrightText: 2025 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from odoo import fields, models
+
+
+class VolunteerShiftRecurrentSubscription(models.Model):
+    _name = "volunteer.shift.recurrent.subscription"
+    _description = "Shift Recurrent Subscription"
+    _order = "start_date"
+    _inherit = [
+        "mail.thread",
+        "mail.activity.mixin",
+    ]
```

Comment:
```suggestion
    ]
    
```
Blank line here to separate _ prefixed var than others.